config.go

```diff
@@ -272,7 +272,7 @@ func loadConfig() (*config, error) {
 
 	// Ensure the fee percentage is valid per txrules.
 	if !txrules.ValidPoolFeeRate(cfg.VSPFee) {
-		return nil, errors.New("invalid vspfee - should be greater than 0.01 and less than 100.0")
+		return nil, errors.New("invalid vspfee - should be 7.5")
 	}
```

Comment:
What does `txrules.ValidPoolFeeRate` do? If that function returns false, is it because vspfee is not 7.5? To rephrase that, if vspfee is 7.5, will `txrules.ValidPoolFeeRate(cfg.VSPFee)` return true? If vspfee is 7.0 (i.e. not 7.5), will `txrules.ValidPoolFeeRate(cfg.VSPFee)` return false?